src/main/resources/warnings.json

```
@@ -6830,7 +6830,7 @@
     "versions": [
       {
         "lastVersion": "4.1",
-        "pattern": ".*"
+        "pattern": "([123][.].+|4[.][0-1])(|[.-].*)"
```

Can be simplified further, but looks fine 😃 

```suggestion
        "pattern": "([123]|4[.][01])(|[.-].*)"
```

---

Thanks for your recommendation Daniel! Needless to say, feel free to use it instead of mine! :-)